### DISCUSSION TITLE
How to debug config issues

### DISCUSSION THREAD

**Discussion by eedgar** [OpenIDC/mod_oauth2#31] [Q&A]
I am trying to figure out how to see the debug logs and figure out what the header settings need to be when there is no separator etc.  see https://goteleport.com/docs/application-access/guides/jwt/   the jwt exists in this header. Teleport-Jwt-Assertion   I am not sure how to make this work with this mod yet.. :/

**Comment by zandbelt** [comment]
you can use `LogLevel oauth2:debug` to see what is happening from the logs

**Comment by chrisinmtown** [comment]
The recommended answer didn't work for me in the version that I'm using.  I resorted to enabling debug-level config for the entire Apache HTTPD server with the following line, and then the oauth2 debug lines finally appeared.
```
LogLevel debug
```
